### PR TITLE
✨ feat: add basic readme

### PR DESCRIPTION
**Pull request by noahjoeris**
This PR adds a simple readme to represent our turtle repo a bit better.

image

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,80 @@
...
+<br />
+<div align="center">
+  <a href="https://turtle.cool">
+    <img src="./app/public/turtle.svg" alt="Logo" width="80" height="80">
+  </a>
+
+  <h3 align="center">Turtle</h3>
+
+  <p align="center">
+    Token transfers done right!
```
Comment (NunoAlexandre) [inline README.md:13]: I like that (I wrote it somewhere originally I believe 😆) but we should use the official tagline:

```suggestion
    Frictionless cross-chain transfers
```

File: README.md [hunk 2]
```diff
@@ -1 +1,80 @@
...
+
+## About The Project
+
+Turtle is your go to app when it comes to cross-chain transfers that leverage fully trustless infrastructure. We aim to provide a unified experience to transfer tokens anywhere.
+
+Features:
+
+- Make transfers between blockchains in a decentralized and trustless way. With a mind blowing UI.
+- Powered by Snowbridge. Transfer ERC20 tokens between Ethereum and Polkadot.
+- Soon to be added: Asset Transfer API support. Transfer tokens seamlessly between all parachains.
```
Comment (NunoAlexandre) [inline README.md:32]:
```suggestion
- Make transfers between blockchains in a decentralized and trustless way 
- Intuitive UI that helps you along the way
- Bridge ERC20 tokens between Ethereum and Polkadot -  Powered by Snowbridge 
- [Soon]: Seamless XCM Transfers between all Polkadot parachains
```

File: README.md [hunk 3]
```diff
@@ -1 +1,80 @@
...
+
+<br/>
+Turtle was developed with Typescript, Nextjs, and React.
```
Comment (NunoAlexandre) [inline README.md:35]:
```suggestion
Turtle is developed with Typescript, Nextjs, and React.
```

File: README.md [hunk 4]
```diff
@@ -1 +1,80 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## Contributing
+
+If you have a suggestion that would make turtle even better, please fork the repo and create a pull request. You can also simply open an issue.
+Don't forget to give the project a star! Thanks again!
```
Comment (NunoAlexandre) [inline README.md:44]:
```suggestion
If you have a suggestion or want to contribute to Turtle, either fork the repo and create a pull request or simply open an issue.

Don't forget to give the project a star. Thanks for your support!
```

File: README.md [hunk 5]
```diff
@@ -1 +1,80 @@
...
+
+Distributed under the MIT License. See `LICENSE` for more information.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTACT -->
+
+## Contact
+
+Nuno - nuno@velocitylabs.org
```
Comment (NunoAlexandre) [inline README.md:74]: I don't mean to be creating hierarchies but it's important to leave this imo:

```suggestion
Nuno (product & tech lead) - nuno@velocitylabs.org
```

**Review by NunoAlexandre — COMMENTED** [review]
This looks beautiful :) left a few comments but thanks already a ton for doing this 🤟

File: README.md (hunk 1 above)
Comment (Victor-Salomon) [reply README.md:13]: Maybe we should update this as well in the app? 
image

**Review by Victor-Salomon — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (NunoAlexandre) [reply README.md:13]: for sure, let's do it in this same PR 👍

File: README.md (hunk 5 above)
Comment (noahjoeris) [reply README.md:76]: 🫡 Sir yes Sir

**Pull request merged**